spark-common/src/main/js/src/spark-bootstrap.js

```diff
@@ -313,19 +440,114 @@ var initIframeAppLoader = function(iframeResizer) {
                 // resizedCallback is called when the iframe is resized which should be enough
                 data.iframe.style.maxHeight = getMaxHeight() + 'px';
             }
-        }, iframeDomEl);
+        };
+
+        // deeply merge the iframeResizer default settings to the iframe options
+        var sparkIframeSettings = AJS.$.extend(true, { 'iframeResizerSettings': iframeResizerSettings }, options);
 
-        return elementIdSparkAppContainer;
+        var { iframeDomEl, iframeSparkContext } = loadSparkToIframe(iframeElement, sparkIframeSettings);
+
+        // Add dialog specific context data
+        iframeSparkContext.dialogControls = {
+            'closeDialog': closeFullScreenDialog,
+            'dialogChrome': dialogChrome
+        };
+
+        return {
```

Comment:
This is a breaking change from previous version, it used to return just the id, now it returns an object with id as res.appContainerId

By design there are no other breaking changes (if there are, they are bugs in the implementation)